### PR TITLE
Fix RunTab contract instance close button and title

### PR DESCRIPTION
**Pull request by ninabreznik**
### 01
Fix RunTab contract instance close button and title
screen shot 2017-10-26 at 6 34 55 pm
---
### 02
Update Terminal menu bar
screen shot 2017-10-27 at 2 03 31 am
---
### 03
Fix left panel menu
screen shot 2017-10-27 at 3 16 25 am
---
 ### 04
Improve terminal table
screen shot 2017-10-29 at 12 27 04 am
---
### 05
Update chat iframe border
screen shot 2017-10-29 at 12 39 01 am
---
### 06 Create copy-to-clipboard module and tooltip module
**!!!! (important: this commit is connected to PR in Remix https://github.com/ethereum/remix/pull/605) !!!!**

screen shot 2017-11-01 at 3 26 55 am

### CODE REVIEW

File: src/app/ui/tooltip.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      top: 100%;
+      left: 50%;
+      margin-left: -5px;
+      border-width: 5px;
+      border-style: solid;
+      border-color: ${styles.remix.tooltip_CopyToClipboard_BackgroundColor} transparent transparent transparent;
+  }
+`
+
+module.exports = function addTooltip (event) {
```
Comment (yann300) [inline src/app/ui/tooltip.js]: can this tool tip be also used when creating a transaction? (for the next PR)

Comment (ninabreznik) [reply src/app/ui/tooltip.js]: @yann300 Sure, we can make a module and reuse it on many places. If you think you would like this, please create an issue and assign me and I will work on it after the IDE.

Comment (yann300) [reply src/app/ui/tooltip.js]: this one https://github.com/ethereum/browser-solidity/issues/911

Comment (yann300) [reply src/app/ui/tooltip.js]: thanks!

**Pull request merged**